register.html

```diff
@@ -1,115 +1,112 @@
-          <a href="#">Withdraw</a>
-          <a href="#">Transfer</a>
-          <a href="#">Account profile</a>
-        </li>
-      </ul>
-
-      <a
-        style="font-size: 30px; cursor: pointer"
-        class="menu"
-        onclick="openNav()"
-        >&#9776;</a
-      >
-    </nav>
-    <div class="container">
-      <form onsubmit="return false;" action="#">
-          <h1>Register</h1>
-
-          <hr />
-
-          <div class="form-group">
-            <label class="form-control-label">Account Name</label>
-            <input
-              type="text"
-              placeholder="Enter name"
-              id="account-name"
-              class="form-control"
-              required
-            />
-          </div>
-
-          <div class="form-group">
-            <label class="form-control-label">Account Number</label>
-            <input
-              type="text"
-              placeholder="Generate account no."
-              maxlength="10"
-              id="account-number"
-              class="form-control"
-              required
-              readonly
-            />
-          </div>
-
-          <div class="form-group">
-            <label class="form-control-label">PIN</label>
-            <input
-              type="password"
-              placeholder="Enter Pin"
-              id="pin"
-              required
-              maxlength="4"
-              class="form-control"
-            />
-          </div>
-
-          <div class="form-group">
-            <label class="form-control-label"><b>Repeat PIN</b></label>
-            <input
-              type="password"
-              placeholder="Repeat Pin"
-              id="pin-repeat"
-              maxlength="4"
-              class="form-control"
-              required
-            />
-          </div>
-
-          <hr />
-
-          <button type="submit" class="registerbtn" class="btn">Register</button>
-      </form>
-    </div>
-
-    <div class="container signin">
-      <p>Already have an account? <a href="#">Sign in</a>.</p>
-    </div>
-
-    <script src="./register.js"></script>
-
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./helpers/components.css" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>MINI BANK APP</title>
+  </head>
+  <body>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()"
+        >&times;</a
+      >
+      <div class="overlay-content">
+        <a href="#">Register</a>
+        <a href="#">Login </a>
+        <a href="#">Transaction history</a>
+        <a href="#">Deposit</a>
+        <a href="#">Withdraw</a>
+        <a href="#">Transfer</a>
+        <a href="#">Account profile</a>
+      </div>
+    </div>
+
+    <!-- <nav>
```

Comment:
Don't comment out, remove completely. If you think you might need the code later or somewhere else, copy the code to a scratchpad.